django_loci/base/models.py

```diff
@@ -62,7 +62,7 @@ def short_type(self):
 
 @python_2_unicode_compatible
 class AbstractFloorPlan(TimeStampedEditableModel):
-    location = models.ForeignKey('django_loci.Location')
+    location = models.ForeignKey('django_loci.Location', on_delete=models.PROTECT)
```

Comment:
the default in django 1.1 was `models.CASCADE` so changing it to protect would be a change in behaviour, which we are not aiming to do now.

Put `models.CASCADE` in place of `models.PROTECT`